AwardsServer/AwardsServer/Database/DatabaseStuffs.cs

```diff
@@ -46,7 +46,13 @@ public void Load_All_Votes()
             // place them into the classes above, then return it.
             // so essentially: this is returning all of the categories, with existing votes already placed into them.
             // (it should also load the AllStudents and AllCategories lists..)
-            connection.Open();
+            try
+            {
+                connection.Open();
+            } catch (Exception ex)
+            {
+                Logging.Log(Logging.LogSeverity.Severe, ex.ToString());
+            }
```

Comment:
In case the file is missing, it'll log the error now

it will error the line after since its not open but hey

Reply:
Good work